doc/project/project_roles.rst

```diff
@@ -296,58 +265,54 @@ Roles / Permissions
 
     ================ =================== =========== ================ =========== =========== ============
 
+Role Retirement
+###############
+
+Individuals approved by the TSC to fill a project role who are no longer actively
+fulfilling the rights and responsibilities associated with their role may be
+requested by the TSC to retire from that role.
+
+Retirements of inactive maintainers or collaborators are reflected by removing the
+individual's GitHub user name from the relevant sections of the
+:ref:`maintainers_file` in the Zephyr repository and may be initiated by the TSC
+or by the individuals themselves.
 
 .. _maintainers_file:
 
 MAINTAINERS File
 ################
 
-Generic guidelines for deciding and filling in the Maintainers' list
-
-* We should keep the granularity of code maintainership at a manageable level
-* We should be looking for maintainers for areas of code that
-  are orphaned (i.e. without an explicit maintainer)
-
-  * Un-maintained areas should be indicated clearly in the MAINTAINERS file
-
-* All submitted pull requests should have an assignee
-* We Introduce an area/subsystem hierarchy to address the above point
-
-  * Parent-area maintainer should be acting as default substitute/fallback
-    assignee for un-maintained sub-areas
-  * Area maintainer gets precedence over parent-area maintainer
+The following guidelines apply to the structure, scope, and maintenance of the
+MAINTAINERS file.
 
-* Pull requests may be re-assigned if this is needed or more appropriate
+- The MAINTAINERS file shall have designated individuals responsible for the
+  accuracy, structure, and upkeep of the file, in accordance with the Zephyr
+  Project Charter. These individuals shall be appointed by the TSC.
+- The granularity of maintainership should remain practical and manageable.
+- The project should actively identify and assign maintainers for orphaned
+  areas of the codebase (i.e., components without an explicit maintainer).
```

Comment:
> actively identify and assign maintainers for orphaned areas of the codebase

This sounds too much like either wishful thinking, or that we will end up be appointing maintainers who will not have the time or ability to do what is needed.
What we can do is be clear about areas not being properly maintained, and try to encourage people to take the role; But not "assign".